tests/phpunit/unit-tests/Controller/ActionTest.php

```diff
@@ -247,7 +245,7 @@ function( $routes ) {
 		/* Fail capability check */
 		try {
 			$this->controller->route();
-		} catch ( Exception $e ) {
+		} catch ( \Exception $e ) {
```

Comment:
We should look at utilising PHPUnit's `@expectException` instead of running the test this way.  https://phpunit.de/manual/6.5/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.exceptions

Reply:
Noted @jakejackson1

Reply:
Applied and use WPDieException